src/main/scala/com/raphtory/output/PulsarOutputFormat.scala

```diff
@@ -7,24 +7,39 @@ import org.apache.pulsar.client.api.Producer
 import org.apache.pulsar.client.api.Schema
 
 /**
-  * # PulsarOutputFormat
-  * `PulsarOutputFormat` writes output to a Raphtory Pulsar topic
+  * {s}`PulsarOutputFormat(pulsarTopic: String)`
+  *   : writes output output to a Raphtory Pulsar topic
+  *        {s}`pulsarTopic: String`
+  *          : Topic name for writing to Pulsar.
+  * Reference: [](com.raphtory.core.algorithm.OutputFormat)
```

Comment:
This is indented wrong and missing blank lines, does not format correctly when compiled. Can you please check the output after fixing this!